Change design for cards

andysellick opened the pull request:
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

## What
- first bit, changing cards in 3 columns on the homepage
- use new branding styles for boxes
- remove link cover as no longer needed
- second bit, reworking progress page
- mainly just adjusting example blocks to match new layout

## Why
New design.

## Visual changes
Before | After
------ | ------
![Screenshot 2024-11-20 at 09 55 17](https://github.com/user-attachments/assets/e93e49ad-6ee8-4e7f-a025-19b133b1fbd4) | ![Screenshot 2024-11-20 at 09 53 01](https://github.com/user-attachments/assets/eb9d29b3-61f1-46b4-a05b-6a5cc8d5fbb9)
![Screenshot 2024-11-20 at 10 45 39](https://github.com/user-attachments/assets/b24d1972-6eb4-4bbd-89eb-7effbfd33e8e) | ![Screenshot 2024-11-20 at 10 45 18](https://github.com/user-attachments/assets/5d6d92f3-d6d0-4f27-af55-122b37fe28ae)

JamesCGDS commented:
Looks good! Just a quick couple of things (and apologies if these are getting a bit too granular for this stage):

- Should the link beneath the cards be styled as bold?
- Do the card headings need some left padding removed on mobile for consistent alignment? (screenshot below)

![image](https://github.com/user-attachments/assets/341ee7f0-3da8-4700-b049-b4925c1813be)

AshGDS reviewed (commented):
Looks good to me, apart the things that @JamesCGDS noticed 👍

matthillco commented:
Looking at the mission helper in [Jon's PR](https://github.com/alphagov/frontend/pull/4416), I think the property names should be made consistent. He's using `mission_type` but you're using `mission` -- that's a bit confusing. Also, since we have the helper and it's merged, would it not be better to use that here? @andysellick @JamesCGDS

andysellick commented:
Thanks @JamesCGDS.

> * Should the link beneath the homepage cards be styled as bold?

Yes, probably. I also need to fix the too-large gap between this and the graphs. Mind if I raise that in a separate PR? Currently not sure how to fix that, but this feels like a step in the right direction.

> * Do the headings on the homepage cards need some left padding removed on mobile for consistent alignment? 

Fixed now, thanks.

> * The non-JS content for the charts on the homepage cards might need updating as they currently say the data is available in a below table

Yes, good shout. I've fixed this, but it's probably dependent on some changes in the component itself, so I'll raise a separate PR for that.

JamesCGDS reviewed (approved):
All looks good to me, nice work! And sounds good re fixing the other stuff in a separate PR
